### PR TITLE
Added ab blocker to telemetry. all so updated the player page to incl…

### PR DESCRIPTION
**Pull request by clint7**
…ude a video with telemetry and ads

### CODE REVIEW

**Comment by sarge** [comment]
Looks pretty good. Build the distribution and version.

File: scripts/shift-player.js [hunk 1]
```diff
@@ -123,6 +123,8 @@ var CastPlayer = function(element, options) {
   this.telemetryTimer = null;
   // used to determine if the media has seeked in the current telemetry window
   this.hasSeeked = false;
+  // @type {Boolean} used to tell if the video has been adblocked
+  this.adBlocker = false;
```
Comment (gscragg) [inline scripts/shift-player.js]: What is the difference between this variable and `adblocked`?

File: scripts/shift-player.js [hunk 2]
```diff
@@ -343,6 +345,7 @@ var CastPlayer = function(element, options) {
 
   function onAdError(adErrorEvent) {
     // Handle the error logging and destroy the AdsManager
+    console.log("adError");
```
Comment (gscragg) [inline scripts/shift-player.js]: Protip:
`console.log('adError', adErrorEvent.getError());`

or `console.error('adError', adErrorEvent.getError());`

File: dist/shift72-player.js [hunk 3]
```diff
@@ -1,167 +1,167 @@
...
-
-    this.chromecast = function(chromecastEnabled){
-      settings.chromecast = !!chromecastEnabled;
-    };
-
-    this.$get = ng.noop;
-  }
-
-}(angular));
+(function(ng){
```
Comment (gscragg) [inline dist/shift72-player.js]: Why has this file changed (built using grunt), but dashplayer.js has not changed?

File: scripts/shift-player.js [hunk 4]
```diff
@@ -887,6 +890,9 @@ CastPlayer.prototype.playMediaLocally = function() {
     if (this.options.adTagUrl && window.google && window.google.ima){
       this.initializeAds(loadPlayer.bind(this));
     } else {
+      if (this.options.adTagUrl){
+        this.adBlocker = true;
```
Comment (gscragg) [inline scripts/shift-player.js]: What does `adBlocker = true` mean? Does that mean the user has an ad blocker?

So ad Blocker is only true, they have an adblocker AND the player is trying to show ads?

Comment (clint7) [inline scripts/shift-player.js]: yes

File: scripts/shift-player.js (hunk 1 above)
Comment (sarge) [inline scripts/shift-player.js]: @clint7 The review process works better when answer all of the questions

File: scripts/shift-player.js (hunk 4 above)
Comment (gscragg) [inline scripts/shift-player.js]: Then shouldn't the variable be named `adsBlocked`?

Comment (clint7) [inline scripts/shift-player.js]: i could be. would you like me to rename it

Comment (gscragg) [inline scripts/shift-player.js]: Yeah, the boolean value is telling us an action happened: Ads were blocked.
Its not telling us the user has an AdBlocker installed (even though it is the same outcome).

Comment (clint7) [inline scripts/shift-player.js]: should i rename the database fields as well. if so should that be new migrations?

Comment (gscragg) [inline scripts/shift-player.js]: My answer would be yes rename the API and DB fields too.

I think you can remove a single migration and add a new one (its only run on your machine). 

Sarge (or maybe google) knows how to do this. Or he might think its not worth it and tell you to leave it as is.

File: scripts/shift-player.js (hunk 1 above)
Comment (clint7) [inline scripts/shift-player.js]: sorry guys. i have rewrote the comment to better suit the variable

**Comment by gscragg** [comment]
Cool :honey_pot:

**Pull request merged**